NEWS.md

```diff
@@ -6,12 +6,14 @@
     * The sign of `sumlargesteigs` has been changed from `Positive` to  `NoSign()` instead of `Positive()`, to allow non-positive-semidefinite inputs. This has the potential
   to break code that required that sign to be positive. If you run into this problem, please file an issue so we can figure out a workaround. [#412](https://github.com/jump-dev/Convex.jl/pull/412)
     * The sign of `eigmin` and `eigmax` has been changed from `Positive` to  `NoSign()`. This is a bugfix because in general `eigmin` and `eigmax` do not need to return a positive quantity (for non-positive-semidefinite inputs). Again, this has the potential to break code that required that sign to be positive. If you run into this problem, please file an issue so we can figure out a workaround. [#412](https://github.com/jump-dev/Convex.jl/pull/413)
```

Comment:
```suggestion
    * The sign of `eigmin` and `eigmax` has been changed from `Positive` to  `NoSign()`. This is a bugfix because in general `eigmin` and `eigmax` do not need to return a positive quantity (for non-positive-semidefinite inputs). Again, this has the potential to break code that required that sign to be positive. If you run into this problem, please file an issue so we can figure out a workaround. [#413](https://github.com/jump-dev/Convex.jl/pull/413)
```